Index: asc-2.2.0.0/source/tools/mount.cpp
===================================================================
--- asc-2.2.0.0/source/tools/mount.cpp
+++ asc-2.2.0.0/source/tools/mount.cpp	2008-12-16 21:25:37.000000000 +0100
@@ -460,7 +460,7 @@
                   fatalError( "could not change directory (1)");
 
                if ( verbose )
-                  printf( direntp->d_name );
+		  printf( "%s", direntp->d_name );
                if ( compress )
                   testcompress ( direntp->d_name,  filesize(direntp->d_name) );
                else {
